.gitignore

```diff
@@ -1,4 +1,2 @@
-/.idea
```

Comment:
These IDE files should be placed in global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore. This is due to prevent putting custom IDE temp files to all git projects.